pkg/systemd/services/services.jsx

```diff
@@ -267,51 +273,61 @@ class ServicesPageBody extends React.Component {
         cockpit.addEventListener("locationchanged", onLocationChanged);
         onLocationChanged(); // initialize
 
-        /* Start listening to signals for updates - when in the middle of reload mute all signals
+        /* Start listening to signals for updates
+         * - when in the middle of reload mute all signals
          * - We don't need to listen to 'UnitFilesChanged' signal since every time we
-         *   perform some file operation we do call Reload which issues 'Reload' signal
-         * - JobNew is also useless, JobRemoved is enough since it comes in pair with JobNew
-         *   but we are interested to update the state when the operation finished
+         *   perform some file operation we do call Reload which issues 'Reloading' signal
          */
         systemd_client[this.props.owner].subscribe({
             interface: "org.freedesktop.DBus.Properties",
             member: "PropertiesChanged"
         }, (path, iface, signal, args) => {
-            if (this.props.isLoading)
+            if (this.props.isLoading || this.reloading)
+                return;
+
+            // ignore uninteresting interfaces and unit types
+            if (args[0] !== "org.freedesktop.systemd1.Unit" && args[0] !== "org.freedesktop.systemd1.Timer")
```

Comment:
I think the timer properties are actually never picked up here, no?

Reply:
They are. If a timer runs, its last/next times get updated:
```
XXX PropertiesChanged on known unit often.timer /org/freedesktop/systemd1/unit/often_2etimer : 
Array(3) [ "org.freedesktop.systemd1.Timer", {…}, (2) […] ]
​
0: "org.freedesktop.systemd1.Timer"
​
1: Object { NextElapseUSecRealtime: {…}, NextElapseUSecMonotonic: {…}, LastTriggerUSec: {…}, … }
​
2: Array [ "TimersMonotonic", "TimersCalendar" ]
```
Without that, the UI wouldn't update.

Reply:
Erk, right! The timer only gets updated on the details page, not the list view. Will fix.

Reply:
Fixed.